src/freva/_user_data.py

```diff
@@ -200,8 +200,7 @@ def delete(self, *paths: os.PathLike, delete_from_fs: bool = False) -> None:
 
         Parameters
         ----------
-
-         *paths: os.PathLike
+        *paths: os.PathLike
```

Comment:
Can you re-add the line here, please?

Reply:
sure, the point was more indentation than the line, as half of the docstrings have a whole line after the Parameters block while others do not. The hrml build seems unnafected by the line.